docs/user/openstack/README.md

```diff
@@ -498,6 +498,14 @@ Another useful thing to check is that the ignition configurations are only avail
 curl https://<loadbalancer ip>:22623/config/master --insecure
 ```
 
+## Refreshing a CA Certificate
+
+If you ran the installer with a custom CA certificate, then this certificate can be changed while the installer is running as needed. To chage your certificate, edit the value of the `ca-cert.pem` key in the `cloud-provider-config` configmap with a valid PEM certificate.
```

Comment:
This should link to the [Self Signed OpenStack CA certificates](https://github.com/openshift/installer/blob/83f2681dbeb943fc9fbe44e8ea8514b2e6362d07/docs/user/openstack/README.md#self-signed-openstack-ca-certificates) section.

I wonder if it wouldn't be better to make a new "Post-Deployment" section and move the External Load Balancer and the Self-signed Cert refresh under it?

And a typo and rewording suggestion:
s/chage/change/
s/can be changed while the installer is running as needed/can be changed later on/

Reply:
I think I prefer "while the cluster is running" just to clarify that this wont blow anything up